recipes/mongodb_org_repo.rb

```diff
@@ -24,15 +24,28 @@
 node.override['mongodb']['package_name'] = 'mongodb-org'
 
 case node['platform_family']
+
 when 'debian'
-  # Adds the repo: http://www.mongodb.org/display/DOCS/Ubuntu+and+Debian+packages
-  apt_repository 'mongodb' do
-    uri "#{node[:mongodb][:repo]}/#{node[:mongodb][:apt_repo]}"
-    distribution 'dist'
-    components ['10gen']
-    keyserver 'hkp://keyserver.ubuntu.com:80'
-    key '7F0CEB10'
-    action :add
+  version = node['mongodb']['package_version'].to_f
+  if version > 2.6 then
+    apt_repository "mongodb-org-#{version}" do
+      uri "http://repo.mongodb.org/apt/#{node['platform']}"
+      distribution "#{node['lsb']['codename']}/mongodb-org/#{version.to_s}"
```

Comment:
If I understand correctly the `node[:mongodb][:repo]` is not used anymore for MongoDB >= 3.0.
What is the rational behind this decision ? Do we expect MongoDB Inc. to have a sane and stable repo naming schema so that we can automatically build the full url even for future versions without requiring any attribute (default or modified by the user) ?
Maybe this could be documented somewhere.
Maybe this could be generalized to 2.6 too.
Maybe we want to still keep a way to override that if something changes in the future, to still let the user capable of fixing it from outside.

If we want both correct default value and possible override, maybe the test on version could be done in the attributes/default.rb like we already do for distribs, or maybe it's not possible due to attributes evaluation order (for distrib it's OK because the use cannot override the distrib), in which case there would be no default value, and all the magic should be here.